<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formulario</class>
 <widget class="QMainWindow" name="Formulario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>733</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>876</width>
    <height>273</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Formulario</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>logo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>521</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h1 align=&quot;center&quot; style=&quot; margin-top:18px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:xx-large; font-weight:700; color:#eeeeec;&quot;&gt;Formul√°rio contato clientes&lt;/span&gt;&lt;/h1&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>751</width>
      <height>369</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_nome">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(211, 215, 207);
</string>
         </property>
         <property name="text">
          <string>Nome:           </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linha_nome">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(186, 189, 182);
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_sobrenome">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(211, 215, 207);</string>
         </property>
         <property name="text">
          <string>Sobrenome:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linha_sobrenome">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(186, 189, 182);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_estudio">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(211, 215, 207);</string>
         </property>
         <property name="text">
          <string>Estudio:        </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linha_estudio">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(186, 189, 182);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_telefone">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(211, 215, 207);</string>
         </property>
         <property name="text">
          <string>Telefone:     </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linha_telefone">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(186, 189, 182);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_email">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(211, 215, 207);</string>
         </property>
         <property name="text">
          <string>Email:            </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linha_email">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(186, 189, 182);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_estado">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(211, 215, 207);</string>
         </property>
         <property name="text">
          <string>Estado:         </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linha_estado">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(186, 189, 182);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_cidade">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(211, 215, 207);</string>
         </property>
         <property name="text">
          <string>Cidade:         </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linha_cidade">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(186, 189, 182);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_logradouro">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(211, 215, 207);</string>
         </property>
         <property name="text">
          <string>logradouro: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linha_logradouro">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(186, 189, 182);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_numero">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(211, 215, 207);</string>
         </property>
         <property name="text">
          <string>numero:        </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linha_numero">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(186, 189, 182);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_cep">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(211, 215, 207);</string>
         </property>
         <property name="text">
          <string>Cep:               </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linha_cep">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(186, 189, 182);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_enviar">
         <property name="styleSheet">
          <string notr="true">background: #259900;</string>
         </property>
         <property name="text">
          <string>Enviar</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
